13-loops/exercises/01-sum-the-numbers/main.go

```diff
@@ -19,4 +13,10 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	var sum int
+
+	for i := 0; i <= 10; i++ {
```

Comment:
it is syntactically correct but try to use i<11 pattern since it gels well with array or slice len() function

Reply:
Fixed to i < 11